src/MICore/LaunchOptions.cs

```diff
@@ -1032,94 +1190,108 @@ public static LaunchOptions GetInstance(HostConfigurationStore configStore, stri
             object launcher = null;
             object launcherXmlOptions = null;
 
-            try
+            if (options.Trim()[0] == '{')
```

Comment:
>(options.Trim() [](start = 15, length = 15)

Might want to move this up so you don't Trim it twice. Ex: change line 1183 to `options = options?.Trim(); if (string.IsNullOrEmpty(options))`